### ISSUE TITLE
Implement <Slider> component support

### ISSUE THREAD

**Issue by MaxRis** [status-im/react-native-linux#87]
[Slider](https://facebook.github.io/react-native/docs/slider.html) component support should be implemented.

**Comment by vkjr** [comment]
@MaxRis, can we consider this issue as `done`?

**Comment by MaxRis** [comment]
@vkjr not really, not all props are implemented. Also it contains child ticket with things to implement.
